functions/src/functions/healthSummary.ts

```diff
@@ -15,11 +16,25 @@ export const exportHealthSummaryFunction = onCall(
     if (!request.data.userId)
       throw new https.HttpsError('invalid-argument', 'User ID is required')
 
-    const service = new HealthSummaryService(
-      new CacheDatabaseService(new FirestoreService()),
+    const databaseService = new CacheDatabaseService(new FirestoreService())
```

Comment:
I am wondering if we should move all of this functionality in a single place to make it easier to compare and reason about and then just call a function of the security service here?

Reply:
If we were to need this functionality somewhere else as well, we can extract it in the SecurityService, ok? I would leave it as is for now